### PR TITLE
votes: add guo-shaoge as tidb committer

### PR DESCRIPTION
**Pull request by XuHuaiyu**
As stated in the title.

### CODE REVIEW

**Review by zimulala — APPROVED** [review]
LGTM

File: votes/0624-guo-shaoge-as-tidb-committer.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# A Vote for windtalker as TiDB committer
```
Comment (tiancaiamao) [inline votes/0624-guo-shaoge-as-tidb-committer.md:1]: s/windtalker/guo-shaoge?

**Review by tiancaiamao — APPROVED** [review]

**Comment by tiancaiamao** [comment]
There's already 2 LGTM
We'll hold on the PR for several days, and if nobody is against this change, it will be merged.

**Pull request merged**